components/kyma-environment-broker/internal/broker/instance_create_test.go

```diff
@@ -25,11 +25,12 @@ import (
 )
 
 const (
-	serviceID       = "47c9dcbf-ff30-448e-ab36-d3bad66ba281"
-	planID          = "4deee563-e5ec-4731-b9b1-53b42d855f0c"
-	globalAccountID = "e8f7ec0a-0cd6-41f0-905d-5d1efa9fb6c4"
-	subAccountID    = "3cb65e5b-e455-4799-bf35-be46e8f5a533"
-	userID          = "test@test.pl"
+	serviceID         = "47c9dcbf-ff30-448e-ab36-d3bad66ba281"
+	planID            = "4deee563-e5ec-4731-b9b1-53b42d855f0c"
+	globalAccountID   = "e8f7ec0a-0cd6-41f0-905d-5d1efa9fb6c4"
+	subAccountID      = "3cb65e5b-e455-4799-bf35-be46e8f5a533"
+	userID            = "Test@Test.pl"
+	userIDToLowerCase = "test@test.pl"
```

Comment:
How exactly it should test something? Is it not better in the first test scenario ("new operation will be created") add to input "upper" `userID` and then check what is the output, if it is lower cases?